package.json

```diff
@@ -35,7 +35,7 @@
     "mkdirp": "^0.5.1",
     "pify": "^3.0.0",
     "tmp": "0.0.29",
-    "uglify-js": "^2.6.2",
+    "uglify-js": "^3.0.21",
```

Comment:
The current plan is to support both `uglify-js` and `uglify-es` to allow users to build both es6 and es5 versions of their code.